### ISSUE TITLE
CI on my fork is failing periodically

### ISSUE THREAD

**Issue by nils-werner** [rustic-rs/rustic#944]
It seems that the periodic nightly publishing CI always runs, indiscriminately whether it's on upstream (here) or my fork.

This leads to a daily [failed run](https://github.com/nils-werner/rustic/actions/runs/6898701860) and a notification in my email inbox.

Is it possible to add a conditional, so that it only runs on upstream?

**Comment by nils-werner** [comment]
No, I don't want to run them, it just seems to be the default when you fork a repo.

**Comment by nils-werner** [comment]
Possibly... but maybe running tests and linting on forks is a good idea, so that people can fix their issues before issuing PRs?

**Comment by aawsome** [comment]
Actually I have to agree with @nils-werner - I would not expect the nightly deployment runs to run on a fork, but I might want to have other CIs running (or not, but then I can deactivate CI for the fork).
I reopen the issue. I can work on it if I find time, but I am also happy I someone can provide a PR!
